server/php-worker.cpp

```diff
@@ -24,6 +24,8 @@
 #include "server/php-worker.h"
 #include "server/server-stats.h"
 
+std::aligned_storage_t<sizeof(PhpWorker), alignof(PhpWorker)> php_worker_storage;
```

Comment:
aligned_storage_t [is deprecated](https://en.cppreference.com/w/cpp/types/aligned_storage)